### DISCUSSION TITLE
Need help for dataset collection.

### DISCUSSION THREAD

**Discussion by Fahad-Siddiqui-ML** [ultralytics/ultralytics#12973] [Q&A]
My aim is to detect HSE related object (i.e. Fire extinguisher, battery cover, driver safety helmet, driving seat belt) on "petrol tank lorry". currently i am in data collection phase for above HSE related object. can anyone help me out for how many images  will i capture? and what resolution ? 
 please find attached sample picture that i collected from MP 4 CCTV camera.
  
  
![kemari terminal_IPC_main_20240520100017_@1](https://github.com/ultralytics/ultralytics/assets/54257342/dc2a9828-c492-4c1d-94e6-d32760144f35)
![kemari terminal_IPC_main_20240520100041_@1](https://github.com/ultralytics/ultralytics/assets/54257342/030d64e5-7cf8-4ef2-9cad-bdf5d3f0ba9c)

**Comment by pderrenger** [comment]
For effective training of a YOLOv8 model for HSE-related object detection on petrol tank lorries, consider the following guidelines:

1. **Image Quantity**: Aim to collect at least 1,000-2,000 images per class for robust training. More images generally lead to better model performance, especially in varied conditions.

2. **Resolution**: The resolution of your training images should be close to the resolution at which you plan to deploy the model. If your CCTV captures at 1080p, it's beneficial to use this resolution for training as well.

Make sure to annotate your images accurately to include all relevant objects. This will help the model learn effectively. Good luck with your dataset collection! 📸

**Reply by Fahad-Siddiqui-ML** [reply]
@pderrenger Thanks !

**Reply by pderrenger** [reply]
@Fahad-Siddiqui-ML you're welcome! If you have any more questions or need further assistance, feel free to ask. Happy coding! 😊
